source/change-streams.rst

```diff
@@ -224,11 +228,32 @@ Driver API
 
   interface Collection {
     /**
+     * @since 3.6
      * @returns a change stream on a specific collection.
      */
     watch(pipeline: Document[], options: Optional<ChangeStreamOptions>): ChangeStream;
   }
 
+  interface Database {
+    /**
+     * Allows a client to observe all changes in a database
+     * @returns a change stream  on all collections in a database
+     * @since 4.0
+     * @note excludes system collections
+     */
+    watch(pipeline: Document[], options: Optional<ChangeStreamOptions>): ChangeStream;
+  }
+
+  interface MongoClient {
+    /**
+     * Allows a client to observe all changes in a cluster
```

Comment:
In the same vein:

> Allows a client to observe changes across all collections and databases within a cluster, excluding system collections and the "config", "local", and "admin" databases.

Likewise, I'd add the same `@see` link below.

Reply:
done